src/us/kbase/common/service/JsonTokenStream.java

```diff
@@ -752,7 +756,12 @@ public JsonToken nextToken() throws IOException, JsonParseException {
 	@Override
 	public JsonToken nextValue() throws IOException, JsonParseException {
 		if (debug) debug();
-		return getInner().nextValue();
+		// copied from com.fasterxml.jackson.core.base.ParserMinimalBase
+		JsonToken t = nextToken();
```

Comment:
You don't need the instance variable in the nextToken() call?

Reply:
Not totally sure I understand what you're asking, but this method doesn't care about the `path` state variable other than it's updated correctly by the `this.nextToken()` call rather than ignored by the `getInner().nextValue()` call.